doc-src/ch-tar2sqfs_desc.rst

```diff
@@ -0,0 +1,44 @@
+Synopsis
+========
+
+::
+
+  $ ch-tar2sqfs TARBALL OUTDIR
+
+Description
+===========
+
+Create Charliecloud squashfs with name matching :code:`TARBALL` under directory :code:`OUTDIR`
+
```

Comment:
Suggest altering `with name matching` to something more consistent with existing descriptions. Perhaps something a long the lines of:

_Create Charliecloud squashfs from `TARBALL` in directory `OUTDIR`. The resulting Squashfs  directory has a name corresponding to `TARBALL` with the `.sqfs` suffix_